packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -1,22 +1,95 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { useBlockProps, PlainText } from '@wordpress/block-editor';
+import { __, _x, isRTL } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	PlainText,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import {
+	PanelBody,
+	__experimentalSegmentedControl as SegmentedControl,
+	__experimentalSegmentedControlOption as SegmentedControlOption,
+} from '@wordpress/components';
 
 export default function QueryPaginationNextEdit( {
-	attributes: { label },
+	attributes: { label, arrow },
 	setAttributes,
 } ) {
+	const arrowControls = [
+		{
+			value: '',
+			label: _x( 'None', 'Arrow option for Query Pagination Next block' ),
+		},
+		{
+			value: isRTL() ? '←' : '→',
+			label: _x(
+				'Arrow',
+				'Arrow option for Query Pagination Next block'
+			),
+		},
+		{
+			value: isRTL() ? '«' : '»',
+			label: _x(
+				'Chevron',
+				'Arrow option for Query Pagination Next block'
+			),
+		},
+	];
+
+	let selectedArrow = arrowControls.filter(
+		( arrowControl ) => arrowControl.value === arrow
+	);
+
+	// This can happen if the user switches from an LTR to RTL site language.
```

Comment:
This can result in loosing the current value and also we can't have `setAttributes` this way as it produces effects. We should have it inside a `useEffect`.

Having said that, I think we should keep a single value and explore  a `css` approach for mirroring them when `isRtl` or checking it when we render the value and change accordingly with a helper `arrowsMapper` object. Either way I think we should also add a `span` wrapper to the arrow for better handling.

I actually tried this approach with the current symbols, but seems it had a bit unexpected results. Not sure if there are some rtl browser implementation for some of them, but for example `«` seems to be mirroring by itself 🤔 

For reference the css I tried is this (added the wrapper in rendering first):
```
.wp-block-query-pagination-next{
	.pagiArrow{
		// Flip for RTL.
		transform: scaleX(1) #{"/*rtl:scaleX(-1);*/"}; // This points the chevron right for LTR and left for RTL.
	}
}
```

Reply:
Yeah, `«` handles it's own RTL!

Reply:
I couldn't find where this is handled? Do you have any resource to share, to read more about it? I added the handling by css for now, but would be good to learn 😄